resources/META-INF/plugin.xml

```diff
@@ -1,394 +1,7 @@
 <idea-plugin url="https://github.com/koxudaxi/pydantic-pycharm-plugin" require-restart="true">
```

Comment:
@koxudaxi Does the plugin need restart? The plugin verify step said that it could be installed without restart

https://github.com/koxudaxi/pydantic-pycharm-plugin/runs/5112235486?check_suite_focus=true#step:10:72
> Plugin can be loaded/unloaded without IDE restart

Reply:
@KotlinIsland 
I forget why I add the restart option 🤔 
If we don't need the option then we should remove it.

Reply:
Seems you changed it with message "Fix build warning", I've removed it now and will see if there are any build warnings.